pokedex/src/Components/App/App.jsx

```diff
@@ -0,0 +1,141 @@
+import React, { act } from "react";
+import Header from "../Header/Header";
+import Pokesprites from "../Pokesprites/Pokesprites";
+import Pokeinfo from "../Pokeinfo/Pokeinfo";
+import Aboutme from "../Aboutme/Aboutme";
+import axios from "axios";
+import { useState } from "react";
+import { useEffect } from "react";
+import "../App/App.css";
+import "../style.css";
+import Footer from "../Footer/Footer";
+import { Routes, Route } from "react-router-dom";
+import pokemonData from "../pokemonapi.json";
+import { useNavigate } from "react-router-dom";
+
+const App = () => {
+  const [loading, setLoading] = useState(true);
+  const [url, setUrl] = useState("https://pokeapi.co/api/v2/pokemon/");
+  const [activePokemon, setActivePokemon] = useState();
+  const [pokeInfo, setPokeInfo] = useState({});
+  const [searchTerm, setSearchTerm] = useState("");
+
+  const [pokemonList, setPokemonList] = useState(pokemonData.results);
+  const filteredPokemonList = pokemonList.filter((pokemon) =>
+    pokemon.name.includes(searchTerm)
+  );
+
+  const pokeData = Object.values(pokeInfo).filter((pokemon) => {
+    const bool = pokemon.name.includes(searchTerm);
+    return bool;
+  });
+
+  const navigate = useNavigate();
+
+  const pokeFun = async () => {
+    setLoading(true);
+    const res = await axios.get(url);
```

Comment:
`axios` is not allowed to be used by the checklist. You need to make requests using `fetch`